proxy/util/cloudsqlutil.go

```diff
@@ -15,7 +15,10 @@
 // Package util contains utility functions for use throughout the Cloud SQL Auth proxy.
 package util
 
-import "strings"
+import (
+    "strings"
+	_ "embed"
```

Comment:
You'll need to run `go fmt ./...` on this code. Let's talk about setting up your favorite editor to do this for you.

Reply:
@annafang-google if you are using vscode I believe just installing the Go official extension will auto run `go fmt` when you save a file.

Reply:
Thank you both! Yes I fixed with go fmt command manually this time. Will integrate it with vim 😄 .